website/blog/2019-01-21-jest-24-new-website-babel-7-and-typescript-support.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Jest 24: New Website, Babel 7 and TypeScript support
+author: Simen Bekkhus
+authorURL: https://github.com/SimenB
+authorFBID: 100003004880942
+---
+
+Today we are happy to announce the next major release of Jest - version 24! It's been 4 months since the last release (and 8 since Jest 23), so this is a meaty one. Highlights include upgrading to Babel 7 across the board (bringing with it built-in support for testing TypeScript) fixing some long standing issues with missing console output and performance issues when computing large diffs, as well as a brand new sparkling website. ✨
```

Comment:
Minor punctuation change and rewording for the sake of our friends who eat vegetarian :)

```suggestion
Today we are happy to announce the next major release of Jest - version 24! It's been 4 months since the last release, and 8 since Jest 23, so this upgrade has something for everyone. Highlights include upgrading Jest to Babel 7 across the board (bringing with it built-in support for testing TypeScript) fixing some long standing issues with missing console output and performance issues when computing large diffs, as well as a brand new sparkling website. ✨
```

Reply:
https://github.com/facebook/jest/pull/7670/commits/86fa69a29a7bba3fb046df2d93b28c3d1329f3b3

yes?